src/createSlfSentryDriver.ts

```diff
@@ -27,7 +29,14 @@ export default function createSlfSentryDriver(
         dsn: sentryUrl,
         tracesSampleRate: 1.0,
         debug: debug ?? ['fat', 'dev'].includes(environment.toLowerCase()),
-        environment
+        environment,
+        integrations: [
+          new Integrations.Http(integrationsOptions?.Http),
+          new Integrations.OnUncaughtException(integrationsOptions?.OnUncaughtException),
+          new Integrations.OnUnhandledRejection(integrationsOptions?.OnUnhandledRejection),
+          new Integrations.ContextLines(integrationsOptions?.ContextLines),
+          new Integrations.InboundFilters(integrationsOptions?.InboundFilters)
```

Comment:
How does this affect the default behaviour?

I think we should only provide the integrations we want to override, e.g. `Http` if we provide the `Http` integration option.

Reply:
https://docs.sentry.io/platforms/node/configuration/integrations/default-integrations/#modifying-system-integrations

Reading the docs, it just says it overrides, and those integrations which isn't in the `integrations` array will just set to default. Can debug and see if that's the case.

Reply:
I debugged this, and it adds everything else which isn't included in the array which it takes it input from.